app/assets/stylesheets/css-conf.scss

```diff
@@ -80,3 +80,12 @@ $f8: 0.75rem;
   --color-gobierto-red: #ff776d;
   --color-gobierto-yellow: #f8b205;
 }
+
+.helper-screenreader {
```

Comment:
Remove this stuff from the configuration file. Create a new one, no matter if it's just this piece of code, and import it from the same place `checkAndReportAccessibility` function is located

Reply:
Done!